packages/utilities/src/initializeFocusRects.ts

```diff
@@ -50,3 +50,23 @@ function _onKeyDown(ev: KeyboardEvent): void {
     }
   }
 }
+
+/**
+ * Sets isFocusVisible
```

Comment:
For the comment, let's be specific about when/why this would be used:

```
Sets the visibility of focus treatments. By default, focus treatments only show up when navigational keypresses occur (through TAB, arrows, pgup/down, home and end), and are hidden when mouse interactions occur. This API provides an imperative way to turn them
on/off. A use case might be when you have a keypress like ctrl-f6 navigate to a particular region on the page, and want focus to show up.